### ISSUE TITLE
Drag Bookmark Folders to Move Them

### ISSUE THREAD

**Issue by luzmcosta** [brave/browser-laptop#10321]
Fix drafted at https://github.com/brave/browser-laptop/commit/315ae5f6ed897c85c1e3c0c8bda692fba01f2fbe.
Let me know how you'd like me to proceed with submitting
this fix given it relies on the code in the open PR #10309.

A user expects to move folders by dragging them. Sadly,
this functionality fails to execute with no user feedback.
That is, the user is able to drag the folders, but dropping
them doesn't move the dragged folder into the new parent
folder, even though the move is valid.

In short, this issue rose because the `folderId` passed
around was not the expected value.

To go into more detail:

The first issue was in the renderer process, which passed
the wrong `folderKey` to the main process method
`moveFolder`. Thus, the `moveFolder` method always led
into the `folder.isEmpty()` condition.

The second issue was later in the `moveFolder` method,
which was not updating the `parentFolderId` property to
the correct folder ID.

**Comment by luixxiul** [comment]
For now I'm going to close this issue as #10309 was closed.

@luzmcosta do you see the issue you discusses on the latest master branch? If so and if you have a fix for that, let's reopen the issue 😄 Thanks!
